docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -149,12 +171,50 @@ When you create a CodeQL database, the extractor stores diagnostic data in the d
 
 If the analysis found fewer results for standard queries than you expected, review the results of the diagnostic and summary queries to check whether the CodeQL database is likely to be a good representation of the codebase that you want to analyze.
 
+Integrating a CodeQL pack into a Code Scanning workflow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. pull-quote::
+
+    Note
+
+    The CodeQL package manager is currently in beta and subject to change. During the beta, CodeQL packs are available only in the GitHub Package Registry (GHPR). You must use version 2.5.8 or later of the CodeQL CLI to use the CodeQL package manager.
+
+Using packs inside your Code Scanning setup allows selecting query packs from various sources.
+
+In the CodeQL configuration file, the section called ``packs`` holds a list of CodeQL package references. Each package listed there will be downloaded by the action and the its default suite will be run.
+
+   queries:
+     - queries/query1.ql
+     - queries/suite1.qls
+   packs:
+     - codeql/pack1@~1.2.3  # latest version compatible with 1.2.3
+     - codeql/pack2         # latest version
+
+For multi-language runs, you can specify a nested map of packs:
+
+   packs:
+     javascript:
+       - codeql/js-pack1@~1.2.3
+       - codeql/js-pack2
+     java:
+       - codeql/java-pack1@~1.2.3
+       - codeql/java-pack2
+
+In the CodeQL workflow file, you can add ``packs`` as input for the ``init`` action by using a comma-separated list of CodeQL packages and optional versions.
+
+   uses: github/codeql-action@v1
+   with:
+     packs: codeql/pack1@~1.2.3, codeql/pack2
```

Comment:
Note that you can prefix with a `+`. It has the same meaning as when you prefix the `queries` input with a `+`. 

With no `+`, the input overwrites the config file. With a `+`, the input is _combined_ with the config file.